.github/workflows/plugin-stat.yml

```
@@ -0,0 +1,28 @@
+name: Pull plugin stats
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
+
+jobs:
+  pull-stats:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 14
```

Maybe use v16? 14 is pretty old 

```suggestion
          node-version: 16.x
```